src/Env.roc

```diff
@@ -48,7 +47,7 @@ var = \name ->
 ## Reads the given environment variable and attempts to decode it.
 ##
 ## The type being decoded into will be determined by type inference. For example,
-## if this ends up being used like a `Task U16 …` then the environment variable
+## if this ends up being used like a `Task U16` then the environment variable
```

Comment:
This was intentional, since task has two type variables...if we need to get rid of the unicode character for some reason, we can use `_` instead of `…`

Reply:
It's been a while, so I can't recall exactly, but I think I removed the unicode characters as they were breaking docs gen at the time. That issue has since been resolved and they now work ok.